commonknowledge/mapped
# Allow choropleth to be toggled on and off

## Description
This PR adds an option to select None in the Type dropdown in the Data Visualisation section to turn off the choropleth

![Untitled](https://github.com/user-attachments/assets/4134b02d-c8e9-4d7d-b0bf-2f569ccc3a1a)

## Motivation and Context
Addresses issue [MAP-711
](https://linear.app/commonknowledge/issue/MAP-711/toggle-choropleth-on-off)

## How Can It Be Tested?
Download the branch and run locally
Create a map or navigate to an existing one
Go to configuration tab and select None under Type in Data Visualisation
Observe that selecting None turns off the choropleth (and reselecting Chropleth toggles it back on again)
